src/gobstuf/api.py

```diff
@@ -115,6 +119,15 @@ def _stuf():
     text = response.text
     text = _update_response(text)
 
+    extra_data = {
```

Comment:
Kunnen we niet twee log messages posten? De eerste als het verzoek wordt ontvangen en de tweede als de response wordt geretourneerd. We weten dan welke verzoeken worden "afgevuurd" op de StUF API en aan wie een antwoord is verzonden.

Reply:
In de extra_data wordt de status_code opgeslagen van de response vanuit de Stuf API. Leek me wat onnodig om twee keer te loggen, aangezien beide events altijd tegelijk gebeuren. Alle informatie in 1 regel heb je ook alles bij elkaar. Volgens mij staat alle nuttige info nu in de log, tenzij ik wat vergeten ben?

Reply:
Als er een illegaal request wordt ontvangen of het request faalt vanwege een exception dan vindt er nu geen logging plaats. Dat lijkt mij niet gewenst.

Reply:
Klopt; maar dat lijkt me niet iets voor in de audit log? > Slack